src/extensions/cp/tools/init.lua

```diff
@@ -1149,7 +1149,7 @@ end
 ---  * path - A path as string
 ---
 --- Returns:
----  * A table containing filenames as strings.
+---  * A table containing filenames as strings, or an empty table if an error occurs.
```

Comment:
This should probably be " or `nil` followed by the error message if an error occurs."